cmd/client-s3-trace_v2.go

```diff
@@ -45,12 +45,18 @@ func (t traceV2) Request(req *http.Request) (err error) {
 		// Set a temporary redacted auth
 		req.Header.Set("Authorization", "AWS **REDACTED**:**REDACTED**")
 
-		var reqTrace []byte
-		reqTrace, err = httputil.DumpRequestOut(req, false) // Only display header
```

Comment:
> With false as last argument. it shouldn't consume so much memory at all.

https://github.com/golang/go/blob/master/src/net/http/httputil/dump.go#L135

For some reason, when false is specified, that code replaces the actual req.Body with a dummy one which has the same content length.